homeassistant/components/smappee/sensor.py

```diff
@@ -79,68 +86,68 @@
     ],
     "solar_today": [
         "Total production - Today",
-        "mdi:white-balance-sunny",
+        None,
         ENERGY_WATT_HOUR,
         "solar_today",
-        None,
+        DEVICE_CLASS_POWER,
```

Comment:
Power sensors should have W or kW as unit. Should this be an energy sensor instead?

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
#54994